### PR TITLE
add message and first commands

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: docs/commands/CorrectTime2000.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+### Parameters
+
+#### Status
+
+`1` - the time setting was successful <br>
+`0` - time setting failed (the [sequence number](#sequence-number) parameter was not changed)
+
+### Examples
+
+success:
```
Comment (DeadByElpy) [inline docs/commands/CorrectTime2000.md:63]:
````
success: ``0x0c 0x01 0x01``

explanation: 
````

Maybe full example more useful?

File: docs/commands/CorrectTime2000.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+ Field        | Value | Dump
```
Comment (DeadByElpy) [inline docs/commands/CorrectTime2000.md:65]: ```Hex        | Value | Field or Part or byte appointment```

?

File: docs/commands/SetTime2000.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+>  1    | byte     | [sequence number](#sequence-number)
+>  4    | int32_be | [seconds](#seconds)
+
+### Parameters
+
+#### **sequence number**
+
+It's a unique number for each time correction request.
+Use the last received [sequence number](./uplink/Time2000.md#sequence-number) + `1` as a starting point.
+Then increment this value for each following request.
```
Comment (DeadByElpy) [inline docs/commands/SetTime2000.md:24]: Increment is not necessary - it can be a pseudo-random value, just need to be unique?

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: docs/commands/CorrectTime2000.md (hunk 2 above)
Comment (DarkPark) [reply docs/commands/CorrectTime2000.md:65]: Dump intentionally is in the end as its size is dynamic

File: docs/commands/CorrectTime2000.md (hunk 1 above)
Comment (DarkPark) [reply docs/commands/CorrectTime2000.md:63]: what do you mean?

the table below contains the full example

Comment (DeadByElpy) [reply docs/commands/CorrectTime2000.md:63]: to use it - need to take it one by one, at this command - it's not difficult, but later it be more complex

File: docs/commands/CorrectTime2000.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/commands/CorrectTime2000.md:65]: not about this, but need to discuss

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**